### PR TITLE
LPS-167985 [Raylife AP] Create Filter and Search to Policies Table

### PR DESCRIPTION
**Pull request by joanagrego**
Related with --> [LPS-158621](https://issues.liferay.com/browse/LPS-158621)

### CODE REVIEW

**Comment by joanagrego** [comment]
ci:test:sf

**Comment by joanagrego** [comment]
si:test:sf

**Comment by joanagrego** [comment]
ci:test:sf

**Comment by joanagrego** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/common/services/Picklists.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+const headlessAPI = 'o/headless-admin-list-type/v1.0/list-type-definitions';
+
+export function getPicklistId(picklistName: string) {
+	return axios.get(
+		`${headlessAPI}/?filter=contains(name, '${picklistName}')&fields=id`
+	);
+}
+
+export function getPolicyStatusPicklist() {
+	const result = getPicklistId('PolicyStatus').then((response) => {
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/common/services/Picklists.ts:26]: Please, change it.
```suggestion
export function getPicklistByName(picklistName) {
	const result = getPicklistId(picklistName).then((response) => {
```

**Review by silasbsi — CHANGES_REQUESTED** [review]
I added one suggestion.

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/common/services/Picklists.ts (hunk 1 above)
Comment (joanagrego) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/common/services/Picklists.ts:26]: Thanks for the suggestion! Done.

**Comment by joanagrego** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/PoliciesTable/index.scss [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+			width: auto;
+		}
+		@media screen and (max-width: 576px) {
+			display: block !important;
+			width: auto;
+		}
+	}
+
+	.hover-button-dropdown {
+		background-color: #f1f2f5 !important;
```
Comment (eliasantosf) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/PoliciesTable/index.scss:19]: Can't **background-color** property be written with Dialect?

**Review by eliasantosf — APPROVED** [review]
Approved! But I left a question.

**Review by silasbsi — APPROVED** [review]

**Comment by silasbsi** [comment]
ci:forward

**Comment by joanagrego** [comment]
ci:forward

**Comment by silasbsi** [comment]
ci:forward

**Comment by joanagrego** [comment]
ci:forward

**Comment by joanagrego** [comment]
ci:forward

**Pull request closed without merging**